components/main/src/test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Main from '.';
+
+describe(Main, () => {
+  let wrapper;
+
+  it('renders without crashing', () => {
+    wrapper = mount(<Main />);
+  });
+
+  it('can render a child node', () => {
+    wrapper = mount(<Main><div>Exmaple</div></Main>);
```

Comment:
Typo in "Example".

Reply:
You can actually get away with `shallow` in these tests since you're just checking for children.

Reply:
Hey, thanks for flagging these, I totally missed these points on the tests - I tested using `shallow` but decided to stick with `mount` as it renders the CSS properties and felt it would be good to have these in the snapshot.